lib/libvarnish/vav.c

```diff
@@ -455,7 +452,7 @@ test_run(const struct test_case *tc, int *ret)
 		return (argv);
 	}
 
-	if (tc->argv[i] != NULL || argv[i] != NULL) {
+	if (tc->argv[0] == NULL && (tc->argv[i] != NULL || argv[i] != NULL)) {
```

Comment:
It would have been nice to see a before/after example in the commit message.

Reply:
updated the commit log